cmd/jaeger/internal/extension/jaegerquery/server.go

```diff
@@ -56,25 +56,11 @@ func (*server) Dependencies() []component.ID {
 
 func (s *server) Start(ctx context.Context, host component.Host) error {
 	var tp trace.TracerProvider
-	success := false
-	tp = jtracer.NoOp().OTEL
+	tp = noop.NewTracerProvider()
+
 	if s.config.EnableTracing {
```

Comment:
Do we still need this setting? Can it not be controlled from telemetry section of otel config?